front/js/app/tools/polygon.js

```diff
@@ -13,6 +13,9 @@ function initPolygon() {
     }
 
     function handleKeydown(event) {
+        if (event.key === 'Escape') {
```

Comment:
This doesn't seem to work for me
I have also tried to substirute condition with `event.keyCode === 27`, but that didn't help

Chromium Version 75.0.3770.90